en/docs/learn/api-security.md

```diff
@@ -64,7 +64,7 @@ This validation allows banks to validate TPPs from the NCAs. This is done by
 validating the transport layer certificate a TPP has obtained. 
 
 ###Role validation 
-According to the [Open Banking Standard - UK](https://www.openbanking.org.uk/about-us/glossary/), TPPs have roles 
+According to the NextGenPSD2 specification, TPPs have roles 
 for the services they provide. The TPP’s role is defined in their transport certificate (QWAC certificate 
 that they obtain from an NCA), which WSO2 Open Banking checks in the API and Dynamic Client Registration flows. 
```

Comment:
Remove "Dynamic Client Registration" part.

Reply:
Fixed in 11987f2244d0bebd44fc743bac5051c8e803e7e7